addons/armaos/XEH_preInit.sqf

```diff
@@ -1,2 +1,19 @@
 #include "script_component.hpp"
-#include "XEH_PREP.hpp"
\ No newline at end of file
+#include "XEH_PREP.hpp"
+
+[
+	"AE3_KeyboardLayout",
+	"LIST",
+	["Keyboard Layout", "Keyboard Layout for armaOS. You can also change this in armaOS terminal."],
+	"AE3 armaOS",
+	[
+		["US", "DE"],
+		[["US", "United States"], ["DE", "Deutschland"]], 
+		0
+	],
+    nil, // "_isGlobal" flag. Set this to true to always have this setting synchronized between all clients in multiplayer
+    {  
+        params ["_value"];
+    }, // function that will be executed once on mission start and every time the setting is changed.
+    true // Setting will be marked as needing mission restart after being changed. (optional, default false) <BOOL>
+] call CBA_fnc_addSetting;
```

Comment:
This is set per mission/server right?

Reply:
All CBA settings are client-side by default as long as `_isglobal == false`, but mission makers and server admins can decide to overwrite client-settings with their mission or server settings, see [here](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System).